packages/web-react/src/components/TextArea/README.md

```diff
@@ -0,0 +1,47 @@
+# TextArea
+
+TextArea enables the user to add longer text to a form. It has textarea, label,
+and an optional message. It could be disabled or have an error state. The label could be hidden
```

Comment:
I'm ok with current component description. 🙂

Reply:
I am a little bit confused about what it means. Are you trying to say that I should show the label when it is required and otherwise it can be hidden? Or that the label can be shown and hidden on demand and when the field is required it is recommended to show the label?

Reply:
Well, if this is my own creation I am not ok with this because I do not understand what I want to tell with that :D